src/StudyEntities/Candidate/EntityType.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+/**
+ * This defines the entity type for a Candidate. It is a Value Object used
```

Comment:
The first part of this sentence is confusing. It sounds like it's saying it's defining a candidate class.